xbmc/GUIPassword.cpp

```diff
@@ -87,6 +90,10 @@ bool CGUIPassword::IsItemUnlocked(T pItem,
         CMediaSourceSettings::GetInstance().UpdateSource(strType, strLabel, "badpwdcount",
                                                          std::to_string(pItem->m_iBadPwdCount));
         CMediaSourceSettings::GetInstance().Save();
+
+        // a mediasource has been unlocked successfully
+        // => refresh favourites due to possible visibility changes
```

Comment:
Differs from code in master
```
// only if this template is instantiated for a CMediaSource item
   if (std::is_same<T, CMediaSource*>::value)
     CServiceBroker::GetFavouritesService().RefreshFavourites();
```

Is there a reason? Looking to get this merged if possible

Reply:
the master PR https://github.com/xbmc/xbmc/pull/20708 needed a fixup (https://github.com/xbmc/xbmc/pull/20781).. this ports both to matrix, thus 2 commits. finally it's equal to master. should i squash and rebase.